src/main/java/se/liu/ida/hefquin/engine/wrappers/lpgwrapper/impl/RegexBasedEdgeLabelMappingToURIsImpl.java

```diff
@@ -0,0 +1,38 @@
+package se.liu.ida.hefquin.engine.wrappers.lpgwrapper.impl;
+
+import org.apache.jena.graph.Node;
+import org.apache.jena.graph.NodeFactory;
+
+public class RegexBasedEdgeLabelMappingToURIsImpl implements EdgeLabelMapping {
```

Comment:
You can make this a subclass that extends `EdgeLabelMappingToURIsImpl`. Then, you don't need to repeat the implementation of the last two methods here, and the `map` method can be implemented to call `map` of the superclass.